lib/ansible/module_utils/redfish_utils.py

```diff
@@ -133,10 +133,14 @@ def _find_systems_resource(self, uri):
             if response['ret'] is False:
                 return response
             data = response['data']
-            for member in data[u'Members']:
-                systems_service = member[u'@odata.id']
-            self.systems_uri = systems_service
-            return {'ret': True}
+            if data[u'Members']:
```

Comment:
Agree that would be a good change.

Reply:
@dtantsur can you please explain, the reason behind it :)
>  We could harden it further by using if data.get(u'Members').

Reply:
@dnuka Perhaps if `Member` key is not present in the `data` dict`, the `data.get()` call would return `None` (which evaluates to `False`) rather than raising `KeyError` exception thus potentially breaking up the work flow.

Reply:
@etingof thank you :)